twoliter/embedded/metadata.spec

```diff
@@ -69,12 +69,25 @@ Provides: %{_cross_os}image-feature(no-encrypted-storage)
 %description
 %{summary}.
 
+%package sbom
+Summary: SBOM metadata for Bottlerocket image
+Requires: %{name}
+Provides: %{_cross_os}image-metadata(sbom)
+
+%description sbom
+SBOM metadata subpackage for Bottlerocket image.
+
 %prep
 
 %build
 
 %install
+mkdir -p %{buildroot}%{_cross_datadir}/bottlerocket
+touch %{buildroot}%{_cross_datadir}/bottlerocket/.sbom-metadata
```

Comment:
We shouldn't need to create this file unless something needs it, we can just have an empty subpackage.